### ISSUE TITLE
Encoding issue

### ISSUE THREAD

**Issue by robrasti** [DiegoZoracKy/convert-excel-to-json#49]
I'm not sure if I'm doing something incorrectly, but I have a spreadsheet that has a few columns as follows:

ContentKey
EnglishText
SpanishText

I am using it to generate a JSON object to force specific Spanish translations.  The problem I'm having is I have in the spreadsheet something like:

Cómo utilizar este informe:

After running through the convert-excel-to-json using Node I end up with the following in my JSON object:

C├│mo utilizar este informe:

Some steps I've taken:
- Saved the excel file specifically with UTF-8 encoding
- Tried reading in the file first, using the fs library using UTF-8 encoding, and passing the buffer to the convert method..

Neither has worked.

This appears to happen with all Spanish specific characters.

Any help would be appreciated if this isn't a bug, but rather user error.

**Comment by robrasti** [comment]
For anyone that stumbled upon this, I was able to resolve the issue, but not in a way I think was very clear.  I was using this as part of a Powershell script that did the following:
- Install npm packages
- run node for a JS file that opened an Excel document, converted it to JSON then returned it as a JSON string using JSON.stringify.
- Redirect that output to a file
- Move that file to a "Data" folder within my project.

Turns out running this in Powershell was causing the character encoding issue somewhere in the process, because when I changed the Powershell scripts over to Bash scripts the problem went away.

I'm not against having my developers run Bash as most have it from installing Git, but it was a frustrating couple hours of my life I will never get back.
